cyberduck/faq.md

```diff
@@ -127,6 +127,16 @@ See _[Preferences → Update](preferences.md#update)_ to switch the update feed.
 - Sophos Anti-Virus *On-access Scanning*
 - Opening the web browser to authenticate using OAuth does not work with [Browser Chooser 2](https://browserchooser2.com/).
 
-## Why does Cyberduck autofil the passwords?
+## Use cases
```

Comment:
I would keep the FAQ short and move this documentation to specific pages such as 

- [ ] https://docs.cyberduck.io/cyberduck/connection/

Reply:
Content was moved to https://docs.cyberduck.io/cyberduck/connection/#problems-connecting